### PR TITLE
1.6.6 release notes

### PR DESCRIPTION
**Pull request by George-Aeillo**

### CODE REVIEW

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -5,7 +5,17 @@ draft: false
 ---
 ## v1.6.6
 #### Enhancements:
-- Added support for Kubernetes v1.27.
+- Added support for Kubernetes v1.27. 
+- Reduced Console Image Size.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:9]: -> console image size. (If you are referring to the Verrazzano console, I should state it; in other words, "Reduced Verrazzano console image size.")

File: content/en/docs/releasenotes/_index.md [hunk 2]
```diff
@@ -5,7 +5,17 @@ draft: false
...
+- Enhanced the Rancher Helm chart to support customizing Rancher pod affinity settings.
+
+#### Fixes:
+- Fixed an issue that prevented tolerating an empty VerrazzanoManagedCluster resource `caSecret` field.
+- Fixed orphaned resources during uninstall when VPO restarts.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:14]: -> during uninstall when the Verrazzano platform operator restarts.

File: content/en/docs/releasenotes/_index.md [hunk 3]
```diff
@@ -5,7 +5,17 @@ draft: false
...
+- Fixed issues where Self-signed CA Certificate Expiry Causes Upgrade to Hang.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:15]: -> self-signed CA certificate (Expiry???) expiration caused the upgrade operation to hang.

File: content/en/docs/releasenotes/_index.md [hunk 4]
```diff
@@ -5,7 +5,17 @@ draft: false
...
+- Fixed issues relating to cert-manager-webhook-OCI upgrade.
+- Fixed issues relating to the Pod Security Policies of prometheus-node-exporter and rancher backup.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:17]: -> pod security policies of prometheus-node-exporter and Rancher backup (or the rancher-backup Operator; not sure what is being referred to here, please check).

File: content/en/docs/releasenotes/_index.md [hunk 5]
```diff
@@ -5,7 +5,17 @@ draft: false
...
+- Fixed issue regarding the update of WebLogic Server cluster resource.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:18]: -> Fixed an issue with updating the WebLogic Server cluster resource.

**Review by rosemarymarano — COMMENTED** [review]
Few edits

File: content/en/docs/releasenotes/_index.md (hunk 5 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:18]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 2 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:14]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 4 above)
Comment (bobdonat) [reply content/en/docs/releasenotes/_index.md:17]: Prometheus Node Exported and Rancher Backup Operator

File: content/en/docs/releasenotes/_index.md (hunk 3 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:15]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 4 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:17]: Thanks!

File: content/en/docs/releasenotes/_index.md [hunk 6]
```diff
@@ -6,16 +6,16 @@ draft: false
 ## v1.6.6
 #### Enhancements:
 - Added support for Kubernetes v1.27. 
-- Reduced Console Image Size.
+- Reduced Verrazzano console Image Size.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:9]: -> image size. (not capitalized)

File: content/en/docs/releasenotes/_index.md [hunk 7]
```diff
@@ -6,16 +6,16 @@ draft: false
...
 #### Fixes:
 - Fixed an issue that prevented tolerating an empty VerrazzanoManagedCluster resource `caSecret` field.
-- Fixed orphaned resources during uninstall when VPO restarts.
-- Fixed issues where Self-signed CA Certificate Expiry Causes Upgrade to Hang.
+- Fixed orphaned resources during uninstall when the Verrazzano platform operator restarts.
+- Fixed issues where self-signed CA certificate expiration caused the upgrade operation to hang.
 - Fixed issues relating to cert-manager-webhook-OCI upgrade.
-- Fixed issues relating to the Pod Security Policies of prometheus-node-exporter and rancher backup.
-- Fixed issue regarding the update of WebLogic Server cluster resource.
+- Fixed issues relating to the Pod Security Policies of Prometheus Node Exporter and Rancher Backup Operator.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:17]: (not sure why all these terms are capitalized)
-> pod security policies of prometheus-node-exporter  and rancher-backup Operator.
(prometheus-node-exporter is the name of the pod; Prometheus Node Exporter is the name of the component, select whichever one is correct.)
(rancher-backup Operator is the proper way to write this operator name, see https://verrazzano.io/latest/docs/backup/rancher/)

**Review by rosemarymarano — COMMENTED** [review]
Few more

File: content/en/docs/releasenotes/_index.md (hunk 6 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:9]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 7 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:17]: Thanks!

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**